tests/callbacks/test_timer.py

```diff
@@ -47,6 +47,13 @@ def on_fit_start(self):
     assert trainer.max_epochs is None
     assert trainer.max_steps is None
 
+    # Make sure max_time still honored even if max_epochs == -1
+    trainer = Trainer(max_time=dict(seconds=1), max_epochs=-1)
```

Comment:
this can just be combined with the test above in line 34, right?

Reply:
Good call